mmv1/templates/terraform/nested_query.go.tmpl

```diff
@@ -243,6 +243,9 @@ func resource{{ $.ResourceName }}ListForPatch(d *schema.ResourceData, meta inter
     return nil, err
   }
 
+{{- if $.CustomCode.PostRead }}
+  {{ $.CustomTemplate $.CustomCode.PostRead false -}}
```

Comment:
the CustomTemplate function is from here: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/resource.go#L1510

the `false` will toggle off the option for appending a new line - a feature we really only cared about for an older migration and should not be relevant either way for you

the `-` tells the go template to delete any whitespace after the statement. Getting an example working with this should show how that ends up looking